src/main/osd/osd.h

```diff
@@ -49,9 +49,9 @@ extern const char * const osdTimerSourceNames[OSD_NUM_TIMER_TYPES];
 #define OSD_CAMERA_FRAME_MIN_HEIGHT 2
 #define OSD_CAMERA_FRAME_MAX_HEIGHT 16    // Rows supported by MAX7456 (PAL)
 
-#define OSD_TASK_FREQUENCY_MIN 30
-#define OSD_TASK_FREQUENCY_MAX 300
-#define OSD_TASK_FREQUENCY_DEFAULT 60
+#define OSD_TASK_FREQUENCY_MIN 1
+#define OSD_TASK_FREQUENCY_MAX 60
+#define OSD_TASK_FREQUENCY_DEFAULT 12
```

Comment:
Why lowering these?

Reply:
Because the OSD task currently runs at 60 Hz, but takes 5 calls to do an update, so refresh rate is actually 12Hz.

The new code starts an update at the given rate (default 12Hz) and then has a variable number of task states to complete the update, each dynamically adjusted in duration to ensure determinism is maintained.